api/auth/model.go

```diff
@@ -71,3 +79,32 @@ type IdpClientConfig struct {
 type IDstruct struct {
 	ID string `json:"id"`
 }
+
+type Session struct {
+	ID           string    `json:"id"`
+	UserID       string    `json:"user_id"`
+	SourceID     string    `json:"source_id"`
+	Domain       string    `json:"domain"`
+	Username     string    `json:"username"`
+	RemoteAddr   string    `json:"remote_addr"`
+	UserAgent    string    `json:"user_agent"`
+	Type         string    `json:"type" jsonschema:"enum=login,enum=sso"`
```

Comment:
You can remove the jsonschema tag as we are not initializing any package for that inside the SDK. The backend should be able to handle that.

Reply:
done